truss/server/common/truss_server.py

```diff
@@ -173,7 +173,6 @@ async def schema(self, model_name: str) -> Dict:
         model: ModelWrapper = self._safe_lookup_model(model_name)
 
         if model.truss_schema is None:
-            print("DID NOT FIND MODEL SCHEMA")
```

Comment:
we should add pre-commit hook to check for these prints.

Reply:
agreed, can do in a follow-up